pkg/controller/realmadmin/settings.go

```diff
@@ -131,6 +124,11 @@ func (c *Controller) HandleSettings() http.Handler {
 			realm.SMSTextTemplate = form.SMSTextTemplate
 		}
 
+		// SMS
```

Comment:
I think we should refactor this controller to have a separate action per tab on the form, can file as a followup issue.